pyproject.toml

```
@@ -26,10 +26,9 @@ description = "Computable object representation and validation for gene fusions"
 license = {file = "LICENSE"}
 dependencies = [
     "pydantic == 2.*",
-    "ga4gh.vrsatile.pydantic ~=0.2.0",
-    "ga4gh.vrs ~=0.8.1",
+    "ga4gh.vrs ~=2.0.0a8",
```

See changes here: https://github.com/ga4gh/vrs-python/compare/2.0.0-a8...2.0.0-a10

I don't think this one will cause any changes

```suggestion
    "ga4gh.vrs ~=2.0.0a10",
```